pkg/validate/container.go

```diff
@@ -99,6 +99,38 @@ var _ = framework.KubeDescribe("Container", func() {
 			testStartContainer(rc, containerID)
 		})
 
+		It("runtime should support pause and unpause container [Conformance]", func() {
+			By("create container")
+			containerID := framework.CreateDefaultContainer(rc, ic, podID, podConfig, "container-for-pause-unpause-test-")
+
+			By("start container")
+			startContainer(rc, containerID)
+
+			By("get container status")
+			Eventually(func() runtimeapi.ContainerState {
+				return getContainerStatus(rc, containerID).State
+			}, time.Minute, time.Second*4).Should(Equal(runtimeapi.ContainerState_CONTAINER_RUNNING))
```

Comment:
Wait a while for the container ready and state transform.

Reply:
LGTM